docs/autofix.md

```diff
@@ -263,3 +264,13 @@ To write the fixed manifest to a different file, use the `--outfile/-o` flag:
 ```
 kubeaudit autofix -f "manifest.yml" -o "fixed.yaml"
 ```
+
+### Using Custom Rules with Kubeaudit Config File
```

Comment:
Now that two commands support the kubeaudit config maybe we can move the documentation from the `all` command (https://github.com/Shopify/kubeaudit/blob/master/docs/all.md#kubeaudit-config) to the home page (https://github.com/Shopify/kubeaudit#configuration-file) and then link there from the `all` and `autofix` readmes?

Reply:
good idea!!!